server/pulp/server/db/model/__init__.py

```diff
@@ -700,3 +700,21 @@ class LazyCatalogEntry(Document):
     checksum = StringField()
     checksum_algorithm = StringField(regex=ALG_REGEX)
     data = DictField()
+
+
+class DeferredDownload(Document):
+    """
+    A collection of units that have been handled by the streamer in the
+    passive lazy workflow that Pulp should download.
+
+    :ivar unit_id:      The associated content unit ID.
+    :type unit_id:      str
+    :ivar unit_type_id: The associated content unit type.
+    :type unit_type_id: str
+    """
+
+    # For backward compatibility
+    _ns = StringField(default='deferred_download')
```

Comment:
May not need the *_ns*.  Check with @bmbouter.  I think he's in the process of getting rid of the need for this in all concrete classes.